BCStudentSoftwareDevTeam/celts
# Volunteer profile admin view issue#5

**Summary:**
Created the Volunteer Profile page (both the Admin and Volunteer View). 
On both pages, we created two tables: one for the list of programs a user showed interest in and the other for displaying all the CELTS programs on a table alongside the  Eligibility, Training Status, and Checkboxes columns. 
Added a modal for the Admin to ban or unban a volunteer from a specific program. The Admin can also edit a volunteer's interest using the checkboxes. When a checkbox is checked off, the page automatically loads and updates the Interest table. 

**To Test:** 
Go to the (IP_Address/profile/username) and make sure that the user is an Admin or a student to see the Admin view or Volunteer View page respectively. Click on the show interest checkboxes and you will see how the page automatically reloads and the Interest Table updated.
For admins to ban or unban a user, click on the edit link and a modal will pop up. Fill in the end date for the ban and indicate the reason for the ban or unban and click the ban/unban button. This will update the checkmark on the web page and also store the notes for the ban/unban in the note table database. 

**Notes:**
We are writing test suites.
